libraries/botbuilder-dialogs-adaptive-testing/src/testActions/customEvent.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TurnContext, TestAdapter, ActivityTypes } from 'botbuilder-core';
+import { TestAction } from '../testAction';
+
+export interface CustomEventConfiguration {
+    name: string;
+    value?: unknown;
+}
+
+export class CustomEvent extends TestAction implements CustomEventConfiguration {
+    public static $kind = 'Microsoft.Test.CustomEvent';
+    /**
+     * The event name.
+     */
+    public name: string;
+
+    /**
+     * Event value.
+     */
+    public value: unknown;
```

Comment:
You could consider extracting a type parameter for this rather than hardcoding `unknown`. Up to you, though.

Reply:
Actually, there is no type of limitation on property `value`.  Is there any good idea?